lib/mpl_toolkits/tests/test_mplot3d.py

```diff
@@ -175,6 +175,30 @@ def test_scatter3d_color():
                color='b', marker='s')
 
 
+@image_comparison(baseline_images=['scatter3d_overlap_markers_yellow'],
+                  extensions=['png'], remove_text=True)
+def test_overlap_markers_yellow_first():
+    fig = plt.figure()
+    x = np.array([-1, 1])
+    y = np.array([1, -1])
+    z = np.array([0, 0])
+    ax = fig.add_subplot(111, projection='3d')
+    patches = ax.scatter(x, y, z, s=3500, c=['b', 'y'])
+    ax.view_init(elev=0, azim=-50)
+
+
+@image_comparison(baseline_images=['scatter3d_overlap_markers_blue'],
+                  extensions=['png'], remove_text=True)
+def test_overlap_markers_blue_first():
+    fig = plt.figure()
+    x = np.array([-1, 1])
+    y = np.array([1, -1])
+    z = np.array([0, 0])
+    ax = fig.add_subplot(111, projection='3d')
+    patches = ax.scatter(x, y, z, s=3500, c=['b', 'y'])
+    ax.view_init(elev=0, azim=130)
+
+
```

Comment:
Can you add a bit more to these tests? Make sure the edge colors are getting sorted properly, too?